ui/components/ui/chip/chip.scss

```diff
@@ -63,4 +63,8 @@
       margin-right: 8px;
     }
   }
+
+  &--max-content {
+    width: max-content;
+  }
```

Comment:
Looks like you still need to remove in the base class?

![Screen Shot 2021-12-01 at 11 47 21 AM](https://user-images.githubusercontent.com/8112138/144303422-ac366c24-c298-40bd-a270-754ab9175050.png)

Reply:
resolved